config-connector/solutions/projects/kpt/project-hierarchy/project.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: resourcemanager.cnrm.cloud.google.com/v1beta1
+kind: Project
+metadata:
+  annotations:
+    # Set this field after finding the folder number.
+    cnrm.cloud.google.com/folder-id: "${FOLDER_NUMBER?}" # {"$ref":"#/definitions/io.k8s.cli.setters.folder-number"}
+  name: project-hierarchy-project # {"$ref":"#/definitions/io.k8s.cli.setters.project-id"}
+spec:
+  name: project-hierarchy-project # {"$ref":"#/definitions/io.k8s.cli.setters.project-name"}
```

Comment:
As discussed earlier, can we have this default to the project-id?

Reply:
There is no way to have it 'default' while preserving it as a separate setter. I've bound it to the project-id setter instead.

Reply:
Ok we opened this issue to investigate: https://github.com/GoogleContainerTools/kpt/issues/430